Lib/test/test_codeop.py

```diff
@@ -277,7 +277,7 @@ def test_filename(self):
     def test_warning(self):
         # Test that the warning is only returned once.
         with warnings_helper.check_warnings(
-                ('"is" with a literal', SyntaxWarning),
+                ('"is" with str literal', SyntaxWarning),
```

Comment:
would it be clearer to have  ``"is" with literal of type 'str'``?

Note that types are often in `:

```
>>> 1+'d'
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'int' and 'str'
```

Reply:
But consistently with only double quotes, or only single quotes?

Reply:
Thanks, I've added quotes to the types. I kept the phrase `'str' literal` instead of `literal of type 'str'`, since I found it a little easier to read and is similar to the way we just mention the types in the add example you give (instead of e.g. saying `value of type 'int' and value of type 'str'`)